docs/onderwerpen/transparantie.md

```diff
@@ -1,41 +1,77 @@
 ---
-title: Transparantie
-summary: Hier staat een korte samenvatting over dit onderwerp
-icon: material/magnify
+title: Transparant zijn over algoritmes
+summary:
 ---
 
-Om openheid te bieden en controleerbaar te zijn moeten overheidsinstanties transparant zijn over inzet van algoritmen en AI-systemen. 
+# Transparant zijn over algoritmes
 
-Het is van belang dat overheden nadenken over hun besluitvormingsprocessen en dat zij de werking en toegevoegde waarde van het inzetten van een algoritme kunnen uitleggen.
-Dit is bijzonder relevant als een algoritme of AI-systeem, al dan niet geautomatiseerd, impact heeft op besluitvorming die burgers raakt.  
+Overheden moeten transparant zijn over hun algoritmes. Dit betekent dat je duidelijke informatie geeft over de algoritmes die je ontwikkelt of gebruikt.
 
-Als burgers geen kennis kunnen nemen van de gebruikte algoritmes en in hoeverre diens output hen raakt, worden ze onrechtmatig beperkt in de mogelijkheid om zicht te verdedigen tegen nadelige gevolgen zoals discriminatie of een onjuist genomen beslissing of besluit.
-Daarnaast versterkt transparantie de controlerende functie van burgers en journalistiek, omdat burgers kunnen aangeven of een uitleg over een algoritmisch systeem duidelijk is en of zij de werking van het systeem hetzelfde ervaren. 
+## Wanneer ben je transparant?
+Je bent transparant over algoritmes en AI als je duidelijk uitlegt:
```

Comment:
```suggestion
Je draagt bij aan transparantie over algoritmes als je duidelijk uitlegt:
```

Reply:
Lastig, want met 'draagt bij' geven we nog maar deels antwoord op de vraag 'Wanneer ben je transparant?' Hoe kunnen we wel het hele antwoord geven? Of zullen we deze alinea anders verder afstemmen in de volgende release?